config/strategies/DEMA.toml

```diff
@@ -1,6 +1,5 @@
-short = 10
-long = 21
```

Comment:
Have you wrote this strategy based on https://www.investopedia.com/articles/trading/10/double-exponential-moving-average.asp ? Because parameters should be long and short IMO

Reply:
Have you read this https://forum.gekko.wizb.it/thread-93.html ? “updates the DEMA strategy to only use a single period n”

Reply:
Yes and I don't know why he wrote this. I think DEMA should have same parameters as MACD, because DEMA supposed to be same thing as MACD but improved. We should wait for a comment from @askmike

Reply:
You can make a PR with DEMA and MACD strategy.

Reply:
@AidasK are you sure? Looking at the first picture in your link:

> This one-minute chart of the e-mini Russell 2000 futures contract shows two different double exponential moving averages; a 55-period appears in blue, a 21-period in pink.

eg. most of the time when people use the DEMA indicator they use multiple. I was assuming [this definition](https://en.wikipedia.org/wiki/Double_exponential_moving_average):

![](https://wikimedia.org/api/rest_v1/media/math/render/svg/c9da4133287d6986ddb38c3c4c3c26635c4b8262)

Reply:
@AidasK If my assumption is wrong (it might just be) let's fix that in a separate PR: this one is the solution to a bounty with specific requirements, if these turn out to be wrong this solution is still valid.

Reply:
@askmike DEMA indicator implementation is correct. It has only one variable. But I was talking about the strategy. Strategy should be configurable with long and short parameters, not only weight. Though, this is just my opinion how it should work.